File: Rivington.IG.Web/Rivington.IG.Domain.Models/Dashboard/InspectionStatusGroupingsView.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text;

namespace Rivington.IG.Domain.Models.Dashboard
{
    public class InspectionStatusGroupingsView
    {
        public InspectionStatusGroupingsView()
        {

        }

        [Key]
        public string StatusId { get; set; }
        public string Status { get; set; }
        public int ZeroToNineteenDays { get; set; }
        public int TwentyToThirtyNineDays { get; set; }
        public int FortyToFiftyNineDays { get; set; }
    }
}

File: Rivington.IG.Web/Rivington.IG.API/src/app/modules/shared/models/dashboard/inspection-status-groupings-view.ts
export class InspectionStatusGroupingsView {
    statusId: string;
    status: string;
    zeroToNineteenDays: string;
    twentyToThirtyNineDays: string;
    fortyToFiftyNineDays: string;

    public constructor(init?:Partial<InspectionStatusGroupingsView>) {
        Object.assign(this, init);
    }
}